database/index.js

```diff
@@ -1,214 +1,149 @@
-  character_id: {
-    type: DataTypes.INTEGER,
-    primaryKey: true,
-    allowNull: false
-  },
-  FK_product_id: {
-    type: DataTypes.INTEGER,
-    allowNull: false
-  },
-  Length: {
-    type: DataTypes.DECIMAL
-  },
-  Fit: {
-    type: DataTypes.DECIMAL
-  },
-  Comfort: {
-    type: DataTypes.DECIMAL
-  },
-  Quality: {
-    type: DataTypes.DECIMAL
+app.get('/reviews', (req, res) => {
+  var productId = req.body.product_id;
+  var page = req.body.page === undefined ? 0:req.body.page;
+  var count = req.body.count === undefined ? 5:req.body.count;
+  var sort = req.body.sort === undefined ? 'newest':req.body.sort;
+  var final = {
+    product: productId,
+    page: page,
+    count: count
+  };
+  var orderLabel;
+  if (sort === 'newest') {
+    orderLabel = [['date', 'ASC']];
+  } else if (sort === 'helpful') {
+    orderLabel = [['helpfulness', 'ASC']];
+  } else if (sort === 'relevant') {
+    orderLabel = [['rating', 'ASC']];
   }
-}, {
-  sequelize: sequelize
-})
-
-const Info = sequelize.define('info', {
-  id: {
-    type: DataTypes.INTEGER,
-    primaryKey: true
-  },
-  product_id: {
-    type: DataTypes.INTEGER
-  },
-  rating: {
-    type: DataTypes.INTEGER
-  },
-  date: {
-    type: DataTypes.BIGINT
-  },
-  summary: {
-    type: DataTypes.TEXT
-  },
-  body: {
-    type: DataTypes.TEXT
-  },
-  recommended: {
-    type: DataTypes.BOOLEAN
-  },
-  reported: {
-    type: DataTypes.BOOLEAN
-  },
-  reviewer_name: {
-    type: DataTypes.TEXT
-  },
-  reviewer_email: {
-    type: DataTypes.TEXT
-  },
-  response: {
-    type: DataTypes.TEXT
-  },
-  helpfulness: {
-    type: DataTypes.INTEGER
-  }
-}, {
-  sequelize: sequelize
-});
-
-const Info_photos = sequelize.define('info_photos', {
-  id: {
-    type: DataTypes.INTEGER,
-    primaryKey: true,
-    allowNull: false
-  },
-  review_id: {
-    type: DataTypes.INTEGER,
-    allowNull: false
-  },
-  url: {
-    type: DataTypes.STRING
-  }
-}, {
-  sequelize: sequelize
-});
-
-const Characteristics_data = sequelize.define('char_data', {
-  id: {
-    type: DataTypes.INTEGER,
-    primaryKey: true,
-    allowNull: false
-  },
-  product_id: {
-    type: DataTypes.INTEGER,
-    allowNull: false
-  },
-  name: {
-    type: DataTypes.STRING
-  }
-}, {
-  sequelize: sequelize
+  Info.findAll({
+    where: {
+      product_id: productId
+    },
+    order: orderLabel,
+    limit: count,
+  })
+    .then((data) => {
+      var promiseOfPhotos = data.map((entry) => {
+        return Info_photos.findAll({
+          where: {
+            review_id: entry.id,
+          }
+        })
+      });
+      Promise.all(promiseOfPhotos)
+        .then((result, err) => {
+          if (err) {
+            throw err;
+          } else {
+          for (var i = 0; i < result.length; ++i) {
+            data[i].dataValues['photos'] = result[i];
+            data[i].dataValues['review_id'] = data[i].dataValues.id;
+            delete data[i].dataValues['id'];
```

Comment:
Is there a benefit to deleting this data?  Is it just for formatting the proper data that the API is expected to send?

Reply:
yup